code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -159,7 +159,7 @@
 /datum/uplink_item/jobspecific/titaniumbroom
 	name = "Titanium Push Broom"
 	desc = "A push broom with a reinforced handle and a metal wire brush, perfect for giving yourself more work by beating up assistants. \
-	When wielded, you will reflect projectiles, and hitting people will have different effects based on your intent."
+			When wielded hitting people will have different effects based on your intent. "
```

Comment:
```suggestion
			When wielded hitting people will have different effects based on your intent. It's also strong enough to deflect projectiles when parrying."
```
Probably let people know that the parry stops bullets.

Reply:
The parry component itself does this automatically.

Reply:
Parry component already does this.

Reply:
Currently nothing else says in the uplink that you can parry with it